guide/raids/amirdrassil.md

```diff
@@ -0,0 +1,445 @@
+
+- Remember to buy {{ site.data.externals.gateway_control_shard }} in Valdrakken. It will allow you to use {{ site.data.externals.demonic_gateway }} easily.
+
+<hr>
+
+# Bosses
+
+<p style="color:red" align="center">Click on a boss banner to expand it.</p>
+
+<div class="dungeon-accordion">
+<div id="accordion">
+<div class="card">
+<div class="card-header" id="gnarlroot">
+<div data-toggle="collapse" data-target="#gnarlroot-collapse" aria-expanded="true" aria-controls="gnarlroot-collapse" class="raid-header amirdrassil_the_dreams_hope1"><h2>Gnarlroot</h2></div>
+</div>
+<div id="gnarlroot-collapse" class="collapse" aria-labelledby="gnarlroot" data-parent="#accordion">
+<div class="card-body" markdown="1">
+
+## Boss Fight Profile: Single Target / AoE
+
+### Talents
+
+#### Lightning T30:
+
+<div class="iframe-holder">
+<iframe src="https://www.raidbots.com/simbot/render/talents/BYQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAoVSSLJUSSLRDRJQiEAAAAAgSAJlkgmASLJpdACIJBBC?width=530&level=70" frameborder="0" width="530px" height="100%">
+</iframe>
+</div>
+
+#### Fire T31:
+
+<div class="iframe-holder">
+<iframe src="https://www.raidbots.com/simbot/render/talents/BYQAAAAAAAAAAAAAAAAAAAAAAAAAAAAg0SrIJtkgDIINKRJtkQOAAAAAAgSAJlkgmASLJppEgEJRgA?width=530&level=70" frameborder="0" width="530px" height="100%">
+</iframe>
+</div>
+
+<hr>
+
+## Fight Breakdown:
+
+#### Damage:
+* Ensure to maximize your output whilst {{ site.data.raid.amirdrassil.uprooted_agony }} is active.
+* Cleave down {{ site.data.raid.amirdrassil.tainted_lasher }} and {{ site.data.raid.amirdrassil.tainted_treant }} (Mythic-only) whenever they are active.
+* Use {{ site.data.talent.swg }} and {{ site.data.talent.gow }} to maintain uptime whilst dealing with {{ site.data.raid.amirdrassil.controlled_burn }} and {{ site.data.raid.amirdrassil.shadowflame_cleave }}.
+
+#### Defensives:
+* Use {{ site.data.talent.as }} or healing consumables during {{ site.data.raid.amirdrassil.potent_fertilization }} before soaking {{ site.data.raid.amirdrassil.splintering_charcoal }} to minimize the damage income of {{ site.data.raid.amirdrassil.ember-charred }}.
+* Use {{ site.data.talent.ag }} during {{ site.data.raid.amirdrassil.uprooted_agony }} to maximize its value and give healers more options to deal damage during this damage amplification.
+* If talented, use {{ site.data.talent.earth_shield }} and {{ site.data.talent.healing_stream_totem }} as a movement filler throughout the fight.
+* Use {{ site.data.spell.healing_surge }} if necessary.
+
+#### Utilities:
+* Use {{ site.data.talent.wrt }} to help players destroy {{ site.data.raid.amirdrassil.doom_roots }} as fast as possible.
+* Use {{ site.data.spell.wind_shear }} to interrupt {{ site.data.raid.amirdrassil.blazing_pollen }} from {{ site.data.raid.amirdrassil.tainted_treant }} (Mythic-only).
```

Comment:
I didn't find Blazing pollen in the yml file and it doesn't appear on the site as a result